### PR TITLE
BREAKING CHANGE: Fix types of XRP values in getServerInfo response

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: src/common/serverinfo.js [hunk 1]
```diff
@@ -49,6 +49,12 @@ function getServerInfo(connection: Connection): Promise<GetServerInfoResponse> {
       reserveIncXrp: 'reserveIncrementXRP',
       seq: 'ledgerVersion'
     });
+    info.validatedLedger.baseFeeXRP =
+      info.validatedLedger.baseFeeXRP.toString();
```
Comment (darkdarkdragon) [inline src/common/serverinfo.js]: Why do tests pass? And I don't see conversion from drop to XRP. What I'm missing?

Comment (clark800) [inline src/common/serverinfo.js]: It was expressed in XRP before, but as a number rather than a string.

Comment (darkdarkdragon) [inline src/common/serverinfo.js]: Argh, my bad. But it is still number in `get-server-info.json`, and test didn't catch this :disappointed:

Comment (clark800) [inline src/common/serverinfo.js]: Ah, nice catch, I can make the tests catch that case.

**Comment by clark800** [comment]
@darkdarkdragon done

**Comment by darkdarkdragon** [comment]
LGTM

**Pull request merged**